lib/engine/game/g_18_ardennes/map.rb

```diff
@@ -365,7 +365,7 @@ def after_lay_tile(hex, tile, entity)
         end
 
         def place_home_token(corporation)
-          # Public companies get their starting tokens by exchange.
+          # Corporations get their starting tokens by exchange.
```

Comment:
There's zero benefit in changing comments. Please leave these unchanged.

Reply:
I do not agree that it is zero benefits in changing comments.
For one thing it is a problem with comments that do not get updated when code change.
For another thing, internally (in the code) corporations, minors and companies are used, so when making the visible part updated there are no reason at all to keep the comment as is.

Reply:
18Ardennes has two types of railway company: minor companies and public companies. These are both implemented as Corporation objects. This comment is explaining the different behaviour between the different type of companies, but if it is changed to read 'Corporations' then it is will be wrong. Some corporations get their starting tokens by exchange, some don't.